appinfo/info.xml

```diff
@@ -34,7 +34,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>4.2.0-alpha.0</version>
+	<version>4.2.1-alpha.0</version>
```

Comment:
```suggestion
	<version>4.2.0-alpha.1</version>
```

main will become 4.2.0. The suffix is for internal versioning during the development phase. 4.2.1 will be the first patch update of 4.2.0

Reply:
Done